provision/acc_provision/acc_provision.py

```diff
@@ -36,13 +33,16 @@
     "openvswitch_version",
 ]
 
-VERSIONS = flavors.VERSIONS
-
-DEFAULT_FLAVOR_OPTIONS = flavors.KubeFlavorOptions
-
-CfFlavorOptions = flavors.CfFlavorOptions
-
-FLAVORS = flavors.FLAVORS
+with open("acc_provision/flavors.yaml", 'r') as stream:
```

Comment:
Can we add an argument to acc-provision which will take a URL for this file? If no argument is provided, the default will be the file at this relative path.